deployment/helm/templates/agent.yaml

```diff
@@ -77,6 +79,9 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       volumes:
+      - name: agent-registration
+        hostPath:
+          path: /var/lib/akri
```

Comment:
this is an ignorance question, but is it strange to have a container create/own a path in /var/lib?  are there any permission issues we could run into?

Reply:
Hmm good point -- more investigation should happen here. I just mimicked device plugin because our agent is just a middle man of sorts to Device Plugin.

Reply:
it doesn't seem entirely uncommon to use this path in a kubernetes application (grafana, zookeeper, cassandra, csi): https://github.com/search?p=2&q=%22%2Fvar%2Flib%22+extension%3Ayaml+mountpath+-virtlet+-heketi+-openvswitch&type=Code

Reply:
kept this as default but made configurable in 00cc473d7bbd8